src/app/missions/detail/detail.component.html

```diff
@@ -155,17 +155,62 @@ <h3>Rewards</h3>
+<ng-container *ngIf="mission && textsLocale">
+  <dl id="chat-bubbles">
+    <div *ngIf="textsLocale.chat_state_1">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_1" [innerHTML]="textsLocale.chat_state_1"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached before the mission is offered.</dd>
+    </div>
+    <div *ngIf="textsLocale.accept_chat_bubble">
+      <dt><blockquote class="chat-bubble" luxTooltip="accept_chat_bubble" [innerHTML]="textsLocale.accept_chat_bubble"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when the mission is accepted.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_2">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_2" [innerHTML]="textsLocale.chat_state_2"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached while the mission is in progress.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_3">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_3" [innerHTML]="textsLocale.chat_state_3"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached when the mission is ready to be completed.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_3_turnin">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_3_turnin" [innerHTML]="textsLocale.chat_state_3_turnin"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.target_objectID"></lux-object> when approached when the mission is ready to be completed.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_4">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_4" [innerHTML]="textsLocale.chat_state_4"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached when the mission is complete.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_4_turnin">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_4_turnin" [innerHTML]="textsLocale.chat_state_4_turnin"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.target_objectID"></lux-object> when approached when the mission is complete.</dd>
+    </div>
+  </dl>
 
-<ng-container *ngIf="text">
-  <app-icon [iconID]="text.turnInIconID"></app-icon>
-</ng-container>
-
-<div *ngIf="textsLocale">
-  <ng-container *ngFor="let text of textsLocale | keys">
-    <h4 [id]="'text_' + text.key">{{text.key}}</h4>
-    <blockquote [innerHTML]="text.value"></blockquote>
+  <ng-container *ngIf="text">
+    <app-icon [iconID]="text.turnInIconID"></app-icon>
   </ng-container>
-</div>
+  <dl>
+    <ng-container *ngIf="textsLocale.completion_succeed_tip">
+      <dt><blockquote luxTooltip="completion_succeed_tip" [innerHTML]="textsLocale.completion_succeed_tip"></blockquote></dt>
```

Comment:
I'd prefer these to have either the keys (i.e. `description`, `in_progress`, ...) or a plain text alternative (e.g. "Shown in the passport", "Shown on the mission progess indicator") outside of the tooltip, just like for the chat bubbles.

Reply:
My plan is to add proper UI to these texts at some point (e.g. the actual LU mission offer dialog for the offer text, but I wanted to do this in a separate PR since it's independent from chat bubbles. I can see though that it would be good to have some text in the meantime. I'll add the plain text description to them for now.